library/include/hipsparse.h

```diff
@@ -5058,6 +5065,56 @@ hipsparseStatus_t hipsparseSpMV(hipsparseHandle_t           handle,
                                 hipsparseSpMVAlg_t          alg,
                                 void*                       externalBuffer);
 
+/* Description: Compute the sparse matrix sparse matrix product */
+HIPSPARSE_EXPORT
```

Comment:
Heads up this function only became available in cuda 11.0. It will fail compilation when using cuda 10.1, 10.2 etc. It needs to be gated with:

#if(!defined(CUDART_VERSION) || CUDART_VERSION >= 11000)
// function
#endif

Similarly in the nvcc hipsparse.cpp file. See my sparseToDense PR.

All of the generic routines I think have this problem. In my sparseToDense hipsparse review I have fixed the other ones I think and I can get that PR to compile on cuda 10.1. Also after getting the compilation to pass I noticed that a bunch of the routines fail the tests but I think that can be fixed in a different task as it happens on a bunch of them.